yangvalidator/settings.py

```diff
@@ -112,12 +112,12 @@
 # Database
 # https://docs.djangoproject.com/en/2.1/ref/settings/#databases
 
-#DATABASES = {
-#    'default': {
-#        'ENGINE': 'django.db.backends.sqlite3',
-#        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
-#    }
-#}
+DATABASES = {
```

Comment:
do we need databases in this project at all? as far as I remeber not.. if not please remove this section completely